hyperspy/signal.py

```diff
@@ -2249,6 +2251,73 @@ def rebin(self, new_shape, out=None):
 
     rebin.__doc__ %= OUT_ARG
 
+    def linear_bin(self, scale, crop=True):
+        """
+        Binning of the spectrum image by a non-integer pixel value.
+
+        Parameters
+        ----------
+        scale: a list of floats for each dimension specify the new:old pixel
```

Comment:
Not sure quite what you mean. This currently follows that same format as the rest of the function parameters in the file?